src/Tempest/View/src/Attributes/ExpressionAttribute.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\View\Attributes;
+
+use Tempest\View\Attribute;
+use Tempest\View\Element;
+use Tempest\View\Elements\PhpDataElement;
+
+final readonly class ExpressionAttribute implements Attribute
+{
+    public function __construct(
+        private string $name,
+    ) {
+    }
+
+    public function apply(Element $element): Element
+    {
+        return new PhpDataElement(
```

Comment:
Maybe it would be cleaner to have a separate `ExpressionElement` as well? Internally it would work exactly the same though.

Reply:
Gonna keep it like this for now